app/models/manageiq/providers/openstack/manager_mixin.rb

```diff
@@ -52,6 +52,7 @@ def openstack_handle(options = {})
       osh
     end
   end
+  alias_method :cinder_handle, :openstack_handle
```

Comment:
This isn't right. The openstack_handle and cinder_handle are not the same thing. The cinder_handle should be a handle to the Cinder service alone, not the global openstack handle.